dev-client/src/components/modals/ActionsModal.tsx

```diff
@@ -113,10 +115,15 @@ export const ActionsModal = forwardRef<
       backgroundColor="grey.200"
       Footer={Footer}
       {...modalProps}>
-      <Heading variant="h5" textAlign="center">
```

Comment:
The design calls for no header text on the logout confirm modal, so I refactored this system to make it optional.